creation/interfaces-and-networks.md

```diff
@@ -692,3 +687,209 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
 > into VMI guest. Because of that, when configuring SR-IOV operator
 > policies, make sure you define a pool of VF resources that uses
 > `driver: vfio`.
+
+Once the operator is deployed, an [SriovNetworkNodePolicy
+](https://github.com/openshift/sriov-network-operator#sriovnetworknodeconfigpolicy)
+must be provisioned, in which the list of SR-IOV devices to expose (with
+respective configurations) is defined.
+
+Please refer to the following `SriovNetworkNodePolicy` for an example:
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-1
+  namespace: sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  mtu: 1500
+  nicSelector:
+    pfNames:
+    - ens1f0
+  nodeSelector:
+    sriov: "true"
+  numVfs: 8
+  priority: 90
+  resourceName: sriov_net
+```
+
+The policy above will configure the `SR-IOV` device plugin, allowing the
+PF named `ens1f0` to be exposed in the SRIOV capable nodes as a resource named
+`sriov_net`.
+
+It will also render an associated `SriovNetworkNodeState`, which looks like:
```

Comment:
This object is present even before the Policy gets created (without the spec part). This line suggests it is created as a reaction to it (or I read it that way)

Reply:
I see; what if I simply indicate "render an associated `SriovNetworkNodeState` **spec**".

Reply:
Hope it's clearer now.

Reply:
Is the State useful for anything anyway? I think we would be safe dropping it. If people would need to troubleshoot, they should go to the operator repo.